contributing/content-style-guide.md

```diff
@@ -529,6 +530,8 @@ dbt Cloud CLI | CLI, dbt CLI
 dbt Core | CLI, dbt CLI
 </div></b>
 
+Make sure you keep the reader close to the content and documentation (second person). For example, to explain that a feature is available to a particular dbt Cloud plan &mdash; use “XYZ is available on Enterprise plans” or “People on Enterprise plans can XYZ..” to keep users closer to the documentation. This will signal to users to check their plan or account status on their own. 
```

Comment:
```suggestion
Make sure you keep the reader close to the content and documentation (second person). For example, to explain that a feature is available to a particular dbt Cloud plan &mdash; use “XYZ is available on Enterprise plans” or “Customers on Enterprise plans can XYZ..” to keep users closer to the documentation. This will signal to users to check their plan or account status on their own. 
```

Reply:
i believe the above rec came from Leona and everyone agreed. also our style guide says to avoid the use of 'customers' so I'm going to keep people to be consistent with the avoidance of using 'customer' and keeping the user closer to the docs. 'customer' already has a layer of separation i believe.